script/install-deps

```diff
@@ -57,6 +57,8 @@ install_grpcwebproxy()
 install_python_libs()
 {
     python3 -m pip install --user setuptools
+    python3 -m pip install --user wheel
+    python3 -m pip install --user grpcio protobuf
```

Comment:
Are these libraries only used by running tests? 
If so, I would suggest installing them in `setup-dev`. 
Only dependencies required by end-users should be installed here.

Reply:
Added them here mainly because `py_unittests` script only calls `install-deps`, not `setup-dev`. I'll move them to `setup-dev` and add the call in `py_unittests` script.

Reply:
@simonlingoogle Adding `setup-dev` to the `py_unittests` script seems to cause that to fail every time. Is there a time limit placed on the action (and causing it to fail because of the added time needed to run `setup-dev`?

Reply:
There shouldn't be a time limit on `setup-dev`.

Reply:
I wonder why log looks fine until it suddenly says exit code 1.

Reply:
Are you able to rerun the tests?

Reply:
Yes

Reply:
Resolved by separating the test dependencies into their own script `install-test-deps`.